fastify.js

```diff
@@ -3,21 +3,10 @@
 const wayfarer = require('wayfarer')
 const urlUtil = require('url')
 const stripUrl = require('pathname-match')
-const fastJsonStringify = require('fast-json-stringify')
-const fastSafeStringify = require('fast-safe-stringify')
-const Ajv = require('ajv')
 const jsonParser = require('body/json')
 
 const supportedMethods = ['GET', 'POST', 'PUT']
-const ajv = new Ajv({ coerceTypes: true })
-
-const payloadSchema = Symbol('payload-schema')
-const querystringSchema = Symbol('querystring-schema')
-const outputSchema = Symbol('output-schema')
-const paramsSchema = Symbol('params-scehma')
-
-const schemas = require('./lib/schemas.json')
-const inputSchemaError = fastJsonStringify(schemas.inputSchemaError)
+const validation = require('./lib/validation')
```

Comment:
we can use the functions directly instead of passing through the validation object.